src/enhancer.js

```diff
@@ -49,9 +49,7 @@ export default function enhanceWithRadium(
 
   // Handle stateless components
   if (isStateless(ComposedComponent)) {
-    ComposedComponent = class extends Component {
-      state: Object;
-
+    ComposedComponent = class extends Component<any, Object> {
```

Comment:
Wasn't sure what's going on here, so I just typed props as `any`. Feel free to improve this type.